.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
-npm-debug.log*
+npm-debug.log
```

Comment:
I think Cris S put in the wildcard because he was getting some npm debug logs with random chars at the end, e.g. `npm-debug.log23123123`.